lua/lazyvim/plugins/extras/coding/codecompanion.lua

```diff
@@ -0,0 +1,47 @@
+return {
+  {
+    "olimorris/codecompanion.nvim",
+    cmd = { "CodeCompanion", "CodeCompanionActions", "CodeCompanionToggle", "CodeCompanionAdd", "CodeCompanionChat" },
+
+    opts = function()
+      local options = require("codecompanion.config")
+      local user = vim.env.USER or "User"
+
+      options.strategies.chat.roles = {
+        llm = "  CodeCompanion",
+        user = "  " .. user:sub(1, 1):upper() .. user:sub(2),
+      }
+
+      options.strategies.chat.keymaps.close.modes = {
+        n = "q",
+        i = "<C-c>",
+      }
+      options.strategies.chat.keymaps.stop.modes.n = "<Esc>"
+      options.strategies.chat.keymaps.send.modes.n = "<CR>"
+
+      return options
+    end,
+
+    keys = {
+      { "<leader>a", "", desc = "+ai", mode = { "n", "v" } },
+      { "<leader>aA", "<cmd>CodeCompanionActions<cr>", mode = { "n", "v" }, desc = "Prompt Actions (CodeCompanion)" },
+      { "<leader>aa", "<cmd>CodeCompanionToggle<cr>", mode = { "n", "v" }, desc = "Toggle (CodeCompanion)" },
+      { "<leader>ac", "<cmd>CodeCompanionAdd<cr>", mode = "v", desc = "Add code to CodeCompanion" },
+      { "<leader>ap", "<cmd>CodeCompanion<cr>", mode = "n", desc = "Inline prompt (CodeCompanion)" },
+    },
```

Comment:
Is it possible to more closely mirror the keys for copilot-chat extra (to make it easier for folks to swap between them)? 

https://github.com/LazyVim/LazyVim/blob/3dbace941ee935c89c73fd774267043d12f57fe2/lua/lazyvim/plugins/extras/coding/copilot-chat.lua#L40-L74

For example:

- `<leader>ap` would be "Prompt Actions" (instead of current `<leader>aA`)

Reply:
Tks for the suggestions, makes a lot of sense.